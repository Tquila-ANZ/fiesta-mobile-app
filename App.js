import React, { Component } from "react";
import { StackNavigator } from "react-navigation";
import { Platform, StyleSheet, Text, View, Image } from "react-native";
import HomeScreen from "./src/containers/HomeScreen";
import CameraScreen from "./src/containers/CameraScreen";
import FormBuilderScreen from "./src/containers/FormBuilderScreen";

//------------------------------------------- Fiesta
// import { StackNavigator } from "./fiesta-packages/tq-anz-navigation";
// import FiestaSuperGrid from "./fiesta-packages/tq-anz-super-grid";

//--------------------------------------------------

export default (App = StackNavigator(
  {
    Home: { screen: HomeScreen },
    Camera: {
      screen: CameraScreen,
      navigationOptions: ({ navigation }) => ({
        title: "Camera"
      })
    },
    FormBuilder: { screen: FormBuilderScreen }
  },
  {
    headerMode: "screen",
    navigationOptions: {
      headerTintColor: "#f9521f",
      headerTitleStyle: {
        color: "#f9521f"
      },
      headerStyle: {
        backgroundColor: "#FFF"
      }
    }
  }
));
