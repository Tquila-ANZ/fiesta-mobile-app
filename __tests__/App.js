import 'react-native';
import React from 'react';
 import App from '../App';
import FiestaBtn from 'fiesta/tq-anz-button';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <App />
  );
});

it('renders correctly', () => {
  const tree = renderer.create(
    <FiestaBtn />
  );
});

test('renders correctly', () => {
  const tree = renderer.create(<FiestaBtn />).toJSON();
  expect(tree).toMatchSnapshot();
});
