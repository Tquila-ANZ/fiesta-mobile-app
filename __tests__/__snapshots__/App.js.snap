// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "marginLeft": 6,
      "marginRight": 6,
    }
  }
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        Object {
          "alignItems": "center",
          "backgroundColor": "#00aec7",
          "flexDirection": "row",
          "height": 70,
          "justifyContent": "center",
        },
      ]
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "fontSize": 24,
        }
      }
    />
  </View>
</View>
`;
